site/docs/composition.md

```diff
@@ -5,4 +5,42 @@ title: Composing Layered & Multi-view Plots
 permalink: /docs/composition.html
 ---
 
-TODO Intro to composition and link to compositional operators
+With Vega-Lite, you cannot only create visualizations that consist of a single view. With Vega-Lite's *view composition algebra*, you can combine views into larger multiview displays. You can combine views through [faceting](facet.html), [layering](layer.html), [concatenation](concat.html), and [repeating](repeat.html).
+
+A view that is composed of multiple views can itself be composed with other views. Through this **hierarchical composiiton**, you can create a whole dashboard as a single specification.
+
+Vega-Lite's compiler infers how input data should be reused across constituent views, and whether scale domains should be unioned or remain independent.
+
+## Faceting
+
+With the `facet` operator, you can partition the data and create a view for each subset to create a [trellis plot](https://en.wikipedia.org/wiki/Small_multiple). As a shortcut, Vega-Lite also has the [`column` and `row` encoding channels](encoding.html#facet) to quickly create a faceted view.
+
+Learn how to use it on the [facet page](facet.html).
+
+## Layering
+
+With the `layer` operator, you can stack multiple views on top of each other. This can be useful to add annotations to views. Vega-Lite automatically unions scale domains and combines axes and legends.
+
+However, Vega-Lite can not enforce that a unioned domain is *semantically meaningful*. To prohibit layering of composite views with incongruent internal structures, the layer operator restricts its operands to be single or layered views.
```

Comment:
Hehe, this was a copy from the Vega-Lite paper. I agree, that it's hard to swallow. I revised it.